kuma/wiki/search.py

```diff
@@ -31,25 +31,25 @@ class WikiDocumentType(document.DocType):
                      'Project_talk:', EXPERIMENT_TITLE_PREFIX]
 
     boost = field.Float(null_value=1.0)
-    content = field.String(analyzer='kuma_content',
-                           term_vector='with_positions_offsets')
-    css_classnames = field.String(analyzer='case_insensitive_keyword')
-    html_attributes = field.String(analyzer='case_insensitive_keyword')
+    content = field.Text(analyzer='kuma_content',
+                         term_vector='with_positions_offsets')
+    css_classnames = field.Keyword()
+    html_attributes = field.Keyword()
     id = field.Long()
-    kumascript_macros = field.String(analyzer='case_insensitive_keyword')
-    locale = field.String(index='not_analyzed')
+    kumascript_macros = field.Keyword()
+    locale = field.Keyword()
     modified = field.Date()
     parent = field.Nested(properties={
         'id': field.Long(),
-        'title': field.String(analyzer='kuma_title'),
-        'slug': field.String(index='not_analyzed'),
-        'locale': field.String(index='not_analyzed'),
+        'title': field.Text(analyzer='kuma_title'),
+        'slug': field.Keyword(),
+        'locale': field.Keyword(),
     })
-    slug = field.String(index='not_analyzed')
-    summary = field.String(analyzer='kuma_content',
-                           term_vector='with_positions_offsets')
-    tags = field.String(analyzer='case_sensitive')
-    title = field.String(analyzer='kuma_title', boost=1.2)
+    slug = field.Keyword()
+    summary = field.Text(analyzer='kuma_content',
+                         term_vector='with_positions_offsets')
+    tags = field.Keyword()
+    title = field.Text(analyzer='kuma_title', boost=1.2)
```

Comment:
Index time boosting is deprecated as per [documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-boost.html)

Reply:
Good catch, I'll update the ``SearchQueryBackend``.